lk2nd/device/dts/msm8226/msm8226-samsung.dts

```diff
@@ -30,5 +30,29 @@
 			};
 		};
 	};
+
+	millet3g {
+		model = "Samsung Galaxy Tab 4 8.0 3G (SM-T331)";
+		compatible = "samsung,millet3g", "samsung,millet";
+		lk2nd,match-bootloader = "T331*";
+
+		lk2nd,dtb-files = "msm8226-samsung-millet3g";
```

Comment:
```suggestion
		lk2nd,dtb-files = "msm8226-samsung-millet3g",
				  "msm8226-samsung-milletwifi";
```

perhaps this assuming the original dtb for milletwifi is compatible with this?

Or just 

```suggestion
		lk2nd,dtb-files = "msm8226-samsung-milletwifi";
```

even for now, assuming you don't need to change anything at all for them both to work on the same dtb

Reply:
I want to separate dtbs. millet-common dtsi, milletwifi and millet3g  dtbs. 
Need i do this changes?